ros/openapi/openapi.json

```diff
@@ -605,11 +608,9 @@
                         "type": "string",
                         "enum": [
                             "Idling",
-                            "Oversized",
```

Comment:
@saltgen Why are we removing `Oversized` state? We're supporting this state currently (But not tested though)

Reply:
@r14chandra AFAIK we don't have an Oversized state atm, for that we will need integration with sources.
So IMO since we don't support it yet there is no need to have it in the docs for now

Reply:
@saltgen - if you remove it here then frontend also needs to be updated. In my opinion let's keep this one.

Reply:
+1 for keeping `oversized` state

Reply:
@kgaikwad Ah yes, makes sense, change pushed.